src/main/java/com/example/hello/HelloRepository.java

```diff
@@ -8,19 +8,17 @@
 
 @Repository
 public class HelloRepository {
-    @Autowired
-    private JdbcTemplate jdbcTemplate;
+	@Autowired
+	private JdbcTemplate jdbcTemplate;
 
-    public Map<String, Object> findById(String id) {
+	public Map<String, Object> findById(String id) {
 
-        // SELECT文
-        String query = "SELECT *" 
-                + " FROM employee" 
-                + " WHERE id=?";
+		// SELECT文
+		String query = "SELECT *" + " FROM employee" + " WHERE id=?";
```

Comment:
こちら１行であれば `+` で繋ぐ必要ないですね。
```suggestion
		String query = "SELECT * FROM employee WHERE id= ?";
```

Reply:
ありがとうございます、確かに！おっしゃる通りです。
「書籍の通り」を意識しすぎて、思考が浅くなっていました。
ありがとうございます。

Reply:
最初は写経大事ですから気になさらず！確認しました！